sway/handlers.c

```diff
@@ -23,8 +23,10 @@ static bool m1_held = false;
 static bool dragging = false;
 static bool m2_held = false;
 static bool resizing = false;
+static bool lock_left, lock_right, lock_top, lock_bottom = false;
 
 static bool floating_mod_pressed(void) {
+	return true;
```

Comment:
This is a temporary hack to allow floating views to resize while we wait for wlc to be patched to allow for proper key checking.